File: 프로그래머스/lv1/12901. 2016년/2016년.py
def solution(a, b):
    num=0
    week = ["THU","FRI","SAT","SUN","MON","TUE","WED"]
    mon=[0,31,29,31,30,31,30,31,31,30,31,30]
    for i in range(0,a) : num+=mon[i]
    return week[(num+b)%7]

File: 프로그래머스/lv1/12901. 2016년/2016년.kt
class Solution {
    fun solution(a: Int, b: Int): String {
        val Week = listOf("THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED")
        val year = listOf(31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31)
        var xDay = (0 until a - 1).map {
            year[it]
        }.sum() + b
        return Week[xDay%7]
    }
}